src/components/Toolbar.tsx

```diff
@@ -205,11 +205,13 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
                     exec(store.dispatch, store.getState, e, { type: 'toolbar' })
                   }}
                 >
-                  <SVG style={{
-                    fill: !isActive || isActive(store.getState) ? fg : 'gray',
-                    width: fontSize + 4,
-                    height: fontSize + 4,
-                  }} />
+                  <SVG
+                    getState={store.getState}
```

Comment:
Is there an alternative to passing 'getState` as a prop? We want to constrain the SVG api.

Reply:
I am not sure but we can add `additionalSVGProps?: (getState: () => State) => any` to Shortcut inferface. Besides this, we need to add additionalProps to `Icon`.
In`toggleSort.tsx` we can decide which Icon we want to render and return a json object like {icon: IconDesc} and we can pass this to Icon (SVG) in `Toolbar.tsx`